returnn/tf/layers/basic.py

```diff
@@ -2759,45 +2759,89 @@ class SplitDimsLayer(_ConcatInputLayer):
   i.e. the input is (batch, time, window * feature),
   you can set axis="F", dims=(window, -1),
   and you will get the output (batch, time, window, feature).
+
+  If the split axis has a dynamic length,
+  exactly one of the axes that we split into need to also have a dynamic length.
+  You can e.g. use this to split the input dimension into smaller "chunks" of a fixed window size.
+  E.g. you could have input (batch, time, feature) and set axis="T", dims=(-1, window),
+  to get output (batch, split_time, window, feature).
+  In this case, the exact sequence lengths are lost and everything is padded to multiples of the window size using
+  the given padding value.
+  Use :class:`ReinterpretDataLayer` to receive back the original sequence lengths after merging.
+
   Also see :class:`SplitBatchTimeLayer`.
+  Also see :class:`MergeDimsLayer` which can undo this operation.
   """
   layer_class = "split_dims"
 
-  def __init__(self, axis, dims, **kwargs):
+  def __init__(self, axis, dims, pad_to_multiples=None, pad_value=0, **kwargs):
     """
     :param str axis: e.g. "F"
     :param tuple[int]|list[int] dims: what the axis should be split into. e.g. (window, -1)
+    :param Optional[bool] pad_to_multiples: If true, will pad along the axis to the next multiple of the remaining dims.
```

Comment:
```suggestion
    :param bool|None pad_to_multiples: If true, will pad along the axis to the next multiple of the remaining dims.
```
Also, from this description, I don't quite understand what this means.